samples/precompiled/HttpSyncStart.cs

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
Thanks for finding this. Actually, the name of the license file is LICENSE, so the header should be:

```
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the MIT License. See LICENSE in the project root for license information.
```